### PR TITLE
refactor: string wrapped bn

### PR DESCRIPTION
**Pull request by GMSteuart**

### CODE REVIEW

File: packages/chain-adapters/src/ethereum/EthereumChainAdapter.ts [hunk 1]
```diff
@@ -277,9 +277,9 @@ export class ChainAdapter implements IChainAdapter<ChainTypes.Ethereum> {
 
     const feeData = (await this.providers.http.getGasFees()).data
     const normalizationConstants = {
-      instant: String(new BigNumber(fees.instant).dividedBy(fees.fast)),
+      instant: new BigNumber(fees.instant).dividedBy(fees.fast).toString(),
```
Comment (0xdef1cafe) [inline packages/chain-adapters/src/ethereum/EthereumChainAdapter.ts:280]: we have a utility called `bnOrZero` that we use everywhere, it just safely wraps `new BigNumber()` to stop runtime errors.

can you please change all the cases of `new BigNumber()` (on lines that you have touched) to use `bnOrZero`?

**Review by 0xdef1cafe — CHANGES_REQUESTED** [review]
see comment about `bnOrZero`

File: packages/chain-adapters/src/ethereum/EthereumChainAdapter.ts (hunk 1 above)
Comment (GMSteuart) [reply packages/chain-adapters/src/ethereum/EthereumChainAdapter.ts:280]: Yes, I can do that.

**Review by 0xdef1cafe — APPROVED** [review]

**Comment by 0xdef1cafe** [comment]
thanks for the changes @GMSteuart <3

**Review by elmutt — APPROVED** [review]

**Review by gomesalexandre — APPROVED** [review]
🍬

**Pull request merged**